### PR TITLE
[dygraph] faster rcnn

### PR DESCRIPTION
**Pull request by liuhuiCNN**
[dygraph] faster rcnn

### CODE REVIEW

File: configs/_base_/readers/faster_fpn_reader.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  # check
+  - RandomFlipImage: {prob: 0.5}
+  - NormalizeImage: {is_channel_first: false, is_scale: true, mean: [0.485,0.456,0.406], std: [0.229, 0.224,0.225]}
+  - ResizeImage: {target_size: 800, max_size: 1333, interp: 1, use_cv2: true}
+  - Permute: {to_bgr: false, channel_first: true}
+  batch_transforms:
+  - PadBatch: {pad_to_stride: 32, use_padded_im_info: false, pad_gt: true}
+  batch_size: 2
+  shuffle: true
+  drop_last: false
```
Comment (jerrywgz) [inline configs/_base_/readers/faster_fpn_reader.yml:16]: set to true

File: configs/_base_/readers/faster_fpn_reader.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+worker_num: 2
```
Comment (jerrywgz) [inline configs/_base_/readers/faster_fpn_reader.yml:1]: fields need to update to im_shape and scale_factor

Comment (heavengate) [inline configs/_base_/readers/faster_fpn_reader.yml:1]: 文件名叫faster_reader就可以了

File: ppdet/modeling/architecture/faster_rcnn.py [hunk 3]
```diff
@@ -13,66 +13,91 @@
...
+            self.inputs, self.rpn_head_out, self.anchor_out)
+        loss_rpn = self.rpn_head.get_loss(rpn_loss_inputs)
+        loss.update(loss_rpn)
+
+        # BBox loss
+        bbox_targets = self.proposal.get_targets()
+        loss_bbox = self.bbox_head.get_loss(self.bbox_head_out, bbox_targets)
+        loss.update(loss_bbox)
+
+        total_loss = fluid.layers.sums(list(loss.values()))
```
Comment (jerrywgz) [inline ppdet/modeling/architecture/faster_rcnn.py:91]: update to paddle.add_n

File: configs/_base_/readers/faster_fpn_reader.yml (hunk 1 above)
Comment (liuhuiCNN) [reply configs/_base_/readers/faster_fpn_reader.yml:16]: done, thanks

File: ppdet/modeling/architecture/faster_rcnn.py (hunk 3 above)
Comment (liuhuiCNN) [reply ppdet/modeling/architecture/faster_rcnn.py:91]: done, thanks.

**Review by heavengate — DISMISSED** [review]

**Pull request closed without merging**